visualization/dashboard/src/Localization/qps-ploc.json

```diff
@@ -0,0 +1,109 @@
+{
+    "selectPoint": "[pj0ZL][©¥Select a point to see its local explanation !!! !!! !!! !!! ]",
```

Comment:
wow what is this language lol

Reply:
pseudo language, makes it easy to spot things that aren't localized (because they wouldn't look weird)

Reply:
interesting